app/src/pages/inside/common/expandedOptions/folder/folder.scss

```
@@ -17,35 +17,51 @@
     position: relative;
 
-    &::before {
-      position: absolute;
-      top: 24px;
-      left: -9px;
-      content: '';
-      display: block;
-      width: 1px;
-      height: calc(100% - 42px);
-      background-color: var(--rp-ui-base-e-100);
+    :global([class*='tree-sortable-item--drop-inside']) & &-title {
```

Stylelint (17.5.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)